draft-ietf-wimse-client-assertion-in-workload-environments.md

```diff
@@ -0,0 +1,546 @@
+---
+title: OAuth 2.0 Client Assertion in Workload Environments
```

Comment:
I think "runtime environments" is a more common name.

Reply:
Let's discuss title in #6